sql/2021/jamstack/adoption_by_region.sql

```diff
@@ -0,0 +1,1250 @@
+    "sub-region": "Western Asia"
+  },
+  "tm": {
+    "name": "Turkmenistan",
+    "region": "Asia",
+    "sub-region": "Central Asia"
+  },
+  "tc": {
+    "name": "Turks and Caicos Islands",
+    "region": "Americas",
+    "sub-region": "Caribbean"
+  },
+  "tv": {
+    "name": "Tuvalu",
+    "region": "Oceania",
+    "sub-region": "Polynesia"
+  },
+  "ug": {
+    "name": "Uganda",
+    "region": "Africa",
+    "sub-region": "Eastern Africa"
+  },
+  "ua": {
+    "name": "Ukraine",
+    "region": "Europe",
+    "sub-region": "Eastern Europe"
+  },
+  "ae": {
+    "name": "United Arab Emirates",
+    "region": "Asia",
+    "sub-region": "Western Asia"
+  },
+  "gb": {
+    "name": "United Kingdom of Great Britain and Northern Ireland",
+    "region": "Europe",
+    "sub-region": "Northern Europe"
+  },
+  "us": {
+    "name": "United States of America",
+    "region": "Americas",
+    "sub-region": "Northern America"
+  },
+  "uy": {
+    "name": "Uruguay",
+    "region": "Americas",
+    "sub-region": "South America"
+  },
+  "uz": {
+    "name": "Uzbekistan",
+    "region": "Asia",
+    "sub-region": "Central Asia"
+  },
+  "vu": {
+    "name": "Vanuatu",
+    "region": "Oceania",
+    "sub-region": "Melanesia"
+  },
+  "ve": {
+    "name": "Venezuela (Bolivarian Republic of)",
+    "region": "Americas",
+    "sub-region": "South America"
+  },
+  "vn": {
+    "name": "Viet Nam",
+    "region": "Asia",
+    "sub-region": "South-Eastern Asia"
+  },
+  "vg": {
+    "name": "Virgin Islands (British)",
+    "region": "Americas",
+    "sub-region": "Caribbean"
+  },
+  "vi": {
+    "name": "Virgin Islands (U.S.)",
+    "region": "Americas",
+    "sub-region": "Caribbean"
+  },
+  "eh": {
+    "name": "Western Sahara",
+    "region": "Africa",
+    "sub-region": "Northern Africa"
+  },
+  "ye": {
+    "name": "Yemen",
+    "region": "Asia",
+    "sub-region": "Western Asia"
+  },
+  "zm": {
+    "name": "Zambia",
+    "region": "Africa",
+    "sub-region": "Eastern Africa"
+  },
+  "zw": {
+    "name": "Zimbabwe",
+    "region": "Africa",
+    "sub-region": "Eastern Africa"
+  },
+  "xk": {
+    "name": "Kosovo",
+    "region": "Europe",
+    "sub-region": "Eastern Europe"
+  }
+};
+return countries[country_code][geo];
+''';
+WITH geo_summary AS (
+  SELECT
+    GET_GEO(country_code, 'region') AS region,
+    IF(device = 'desktop', 'desktop', 'mobile') AS client,
+    origin,
+    COUNT(DISTINCT origin) OVER (PARTITION BY GET_GEO(country_code, 'region'), IF(device = 'desktop', 'desktop', 'mobile')) AS total
+  FROM
+    `chrome-ux-report.materialized.country_summary`
+  WHERE
+    # We're intentionally using May 2021 CrUX data here.
+    # That's because there's a two month lag between CrUX and HA datasets.
+    # Since we're only JOINing with the CrUX dataset to see which URLs
+    # belong to different countries (as opposed to CWV field data)
+    # it's not necessary to look at the 202107 dataset.
+    yyyymm = 202105
+)
+SELECT
```

Comment:
```suggestion
''';

WITH geo_summary AS (
  SELECT
    GET_GEO(country_code, 'region') AS region,
    IF(device = 'desktop', 'desktop', 'mobile') AS client,
    origin,
    COUNT(DISTINCT origin) OVER (PARTITION BY GET_GEO(country_code, 'region'), IF(device = 'desktop', 'desktop', 'mobile')) AS total
  FROM
    `chrome-ux-report.materialized.country_summary`
  WHERE
    # We're intentionally using May 2021 CrUX data here.
    # That's because there's a two month lag between CrUX and HA datasets.
    # Since we're only JOINing with the CrUX dataset to see which URLs
    # belong to different countries (as opposed to CWV field data)
    # it's not necessary to look at the 202107 dataset.
    yyyymm = 202105
)

SELECT
```